libs/libcommon/pyproject.toml

```diff
@@ -11,7 +11,7 @@ appdirs = "^1.4.4"
 cryptography = "^42.0.4"
 datasets = {version = "^2.18.0", extras = ["audio", "vision"]}
 environs = "^9.5.0"
-huggingface-hub = "^0.21.3"
+huggingface-hub = {extras = ["hf-transfer"], version = "^0.21.4"}
```

Comment:
I think we have to run `poetry update libcommon` in all the "projects" that depend on it, to update their poetry.lock files